third_party/phantomjs/phantomjs.sh

```diff
@@ -26,6 +26,13 @@ if [[ "${RUNFILES}" == "" ]]; then
   fi
 fi
 
+# Detect if phantomjs is running within rules_closure or has been imported
+# into another project
+PHANTOMJS_BIN_PATH="third_party/phantomjs/bin/phantomjs"
+if ! [ -e ${RUNFILES}/${PHANTOMJS_BIN_PATH} ]; then
+	PHANTOMJS_BIN_PATH=external/io_bazel_rules_closure/${PHANTOMJS_BIN_PATH}
+fi
+
 export LD_LIBRARY_PATH="${RUNFILES}/third_party/fontconfig/k8:${LD_LIBRARY_PATH}"
```

Comment:
Why isn't it necessary to use the external/io_bazel_rules_closure prefix on these too?